content/en/quickstart/device-to-device.md

```diff
@@ -11,3 +11,428 @@ menu:
     weight: 20
 toc: true
 ---
+This guide will walk you through the process of discovering, on-boarding, controlling your secure [OCF Device](https://openconnectivity.org/specs/OCF_Device_Specification_v2.2.3.pdf) using **plgd/sdk** library on your PC.
+
+## Prerequisite
+- The Cloud Server runs on your Raspberry Pi
+
+## Install OCF Client example
+1. Checkout [plgd/sdk](https://github.com/plgd-dev/sdk/tree/v2) library:
+    ```shell script
+    # execute commands on your PC
+    git clone -b v2 https://github.com/plgd-dev/sdk.git --recursive
+    cd sdk
+    ```
+2. Build OCF Client example:
+    ```shell script
+    cd cmd/ocfclient
+    go build
+    ```   
+
+3. Run OCF Client example in the same folder:
+    ```shell script
+    ./ocfclient 
+    ```
+Now, the OCF Client is running on your PC to discover your device and so on. More information about full application sources can be found [here](https://github.com/plgd-dev/sdk/tree/v2/cmd/ocfclient).
```

Comment:
The OCF Client is now running on your PC. It's a console application that allows you to communicate with the device on your local network, without the plgd cloud. Code is available [here]...